test/micro/org/openjdk/bench/jdk/classfile/Write.java

```diff
@@ -65,7 +65,17 @@
  */
 @Warmup(iterations = 3)
 @Measurement(iterations = 5)
-@Fork(1)
+@Fork(value = 1, jvmArgsAppend = {
+        "--enable-preview",
+        "--add-exports", "java.base/jdk.internal.org.objectweb.asm=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.org.objectweb.asm.tree=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.classfile=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.classfile.attribute=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.classfile.constantpool=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.classfile.instruction=ALL-UNNAMED",
+        "--add-exports", "java.base/jdk.internal.classfile.java.lang.constant=ALL-UNNAMED",
```

Comment:
Since this package is gone, this export arg can be removed. Same for other benchmarks.

Reply:
Good catch, thanks!